frontend/src/metabase/query_builder/components/AggregationWidget.jsx

```diff
@@ -58,7 +56,7 @@ export default class AggregationWidget extends React.Component {
         }
       >
         <AggregationLabel className={className}>
-          {isRows(aggregation) ? t`Raw data` : aggregation.displayName()}
+          {aggregation.displayName()}
```

Comment:
@daltojohnso and @ranquild E2E test in `admin` group is failing because metrics page is broken. Rerunning doesn't help. I can confirm it's failing locally as well.
![scenarios  admin  datamodel  metrics -- with no metrics -- custom expression aggregation should work in metrics (metabase#22700) (failed)](https://user-images.githubusercontent.com/31325167/196222148-54d04930-d6fd-43e8-9e90-bc53a5e56604.png)

To reproduce:
1. Go to `/admin/datamodel/metrics`
2. Click on "New metric"
3. Click on "Select a table"
4. Select any table and see the whole page crash with "Something went wrong" message

```
AggregationWidget.jsx:58 Uncaught TypeError: aggregation.displayName is not a function
```

Reply:
@nemanjaglumac fixed